src/styles/style.css

```diff
@@ -0,0 +1,100 @@
+@import "/styles/stars.css";
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0 ;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
card need to be in the `card.css` 😉

Reply:
<img width="565" alt="image" src="https://user-images.githubusercontent.com/55802630/173080337-1151ac88-b6c3-4999-b37d-d49d0195a176.png">